### PR TITLE
General maintenance 001

### PR DESCRIPTION
**Pull request by codeGlaze**
## Description:
**For team review...**
I "normalized" the `project.clj` again for us windows users (all active members atm).

I started snipping out all references to the FB login logic and code.

I'm not sure whether this should be check in or if this could be considered a preview for your comments? Anyhoo, here it is.

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] I have commented my code, particularly in hard-to-understand areas
  - [x] I have made corresponding changes to the documentation if nessecary
  **- [ ] There is a lot of commented code in this PR**
  - [x] My changes generate no new warnings (check the console)

### CODE REVIEW

**Comment by codeGlaze** [comment]
I'm not sure this is error free. I get a `"E" #object[TypeError TypeError: Cannot read property 'call' of undefined]` when logging in.

But it doesn't seem to hurt anything?

File: src/cljs/orcpub/dnd/e5/views.cljs [hunk 1]
```diff
@@ -4374,6 +4378,236 @@
           #(dispatch [toggle-event :tool-prof-or-expertise key])]])
       equip/tools))]])
 
+;
```
Comment (datdamnzotz) [inline src/cljs/orcpub/dnd/e5/views.cljs:4381]: Can these be moved to another branch?

File: src/cljs/orcpub/dnd/e5/views.cljs [hunk 2]
```diff
@@ -5573,6 +5807,13 @@
        ::e5/edit-class-selection-num
        ::e5/edit-class-selection-level
        ::e5/delete-class-selection]]
+     [:div [class-starting-equipment class]] ;codeglaze
```
Comment (datdamnzotz) [inline src/cljs/orcpub/dnd/e5/views.cljs:5810]: Moved to another branch.

File: resources/public/css/compiled/styles.css [hunk 3]
```diff
@@ -1 +1 @@
...
...9px){.hidden-md{display:none !important}}@media(min-width:1200px){.hidden-lg{display:none !important}}@media print{.visible-print{display:block !important}th.visible-print,td.visible-print{display:table-cell !important}table.visible-print{display:table !important}tr.visible-print{display:table-row !important}}.visible-print-block{display:none !important}@media print{.visible-print-block{display:block !important}}.visible-print-inline{display:none !important}@media print{.visible-print-inline{display:inline !important}}.visible-print-inline-block{display:none !important}@media print{.visible-print-inline-block{display:inline-block !important}}@media print{.hidden-print{display:none !important}}@media(max-width:767px){.user-icon{display:none}.list-character-summary{font-size:18px}.character-summary{flex-wrap:wrap}.app-header{height:auto;background-image:none;background-color:rgba(0,0,0,0.3);min-height:0}.app-header-bar{min-height:50px;backdrop-filter:none;-webkit-backdrop-filter:none}.content{width:100%}.header-button-text,.header-links{display:none}}@media(max-width:1199px){.registration-image{display:none}.registration-content{width:100%;height:100%}.registration-input{width:100%}}
```
Comment (datdamnzotz) [inline resources/public/css/compiled/styles.css:1]: This file just needs to be removed from source, it is auto built.

**Review by datdamnzotz — CHANGES_REQUESTED** [review]
Lets break out the FB removal, formatting and, weapons mod into their own branches.

File: resources/public/css/compiled/styles.css (hunk 3 above)
Comment (codeGlaze) [reply resources/public/css/compiled/styles.css:1]: So should we add it to the gitignore or just delete it?

Comment (datdamnzotz) [reply resources/public/css/compiled/styles.css:1]: Just remove it from tracked code...

**Pull request closed without merging**